pages/common/meteor.md

```diff
@@ -0,0 +1,27 @@
+# meteor
+
+> Full-stack javascript platform for building web applications.
+
+- Run a meteor project from its root directory in development mode:
+
+`meteor`
+
+- Create a project under the given directory:
+
+`meteor create {{path/to/directory}}`
+
+- Display the list of packages your project is currently using:
+
+`meteor list`
+
+- Add a package to your project:
+
+`meteor add {{package_name}}`
+
+- Remove a package from your project:
+
+`meteor remove {{package_name}}`
+
+- Create a production build of your project under the given directory. (Omit the --directory option if you want to wrap the build inside a tarball instead):
```

Comment:
> Also, the second part could be a separate example. It would make this page longer than we've been comfortable with so far, 

Yea more than 6 examples is a bit too much. I will just remove the '--directory' option. Since no other commands have a repeat example, its better to keep things tidy.